Lib9c/TableData/Sheet.cs

```diff
@@ -119,7 +119,9 @@ public void Set(string csv, bool isReversed = false)
 
         public void Set<T>(Sheet<TKey, T> sheet, bool executePostSet = true) where T : TValue, new()
         {
+#pragma warning disable LAA1002
             foreach (var sheetRow in sheet)
```

Comment:
Is it okay we don't mind the order of `AddRow()`?

Reply:
yes, beacuse sorting in `PostSet`.

https://github.com/planetarium/lib9c/blob/9fa307cd4cd9fbd4496ee50c714b703c380184c9/Lib9c/TableData/Sheet.cs#L160-L170